lib/core/engine/command/measure.js

```diff
@@ -61,9 +61,18 @@ class Measure {
 
   // Have a concistent way of starting the video
   async _startVideo(numberOfMeasuredPages, index) {
-    this.video = new Video(this.storageManager, this.options);
-    await setOrangeBackground(this.browser.getDriver(), this.options);
-    await this.video.record(numberOfMeasuredPages, index);
+    this.video = new Video(this.storageManager, this.options, this.browser);
+
+    if (this.options.firefox.windowRecorder) {
```

Comment:
Awkward.  Any reason to not set orange before _and_ after we start recording?  IIUC, the existing code will look for orange... is it possible that an orange-to-orange transition will look like a first paint to the existing system?

Reply:
Unfortunately setting the background orange will insert a <div> element, so doing this twice inserts 2 elements we need to remove later on and that can add unnecessary delay.  Not sure if there's any non-awkward solution for this unless we can have Firefox emit the current frame on recording start.